src/main/scala/edu/ie3/mobsim/model/ElectricVehicle.scala

```diff
@@ -473,10 +432,9 @@ case object ElectricVehicle extends LazyLogging {
       chargingAtHomePossible = isChargingAtHomePossible,
       chosenChargingStation = None,
       chargingAtSimona = false,
-      finalDestinationPoiType = None,
       finalDestinationPoi = None,
       remainingDistanceAfterChargingHub = None,
-      chargingPricesMemory = mutable.Queue[Double]()
+      chargingPricesMemory = immutable.Queue[Double]()
```

Comment:
Nicer like this (since you already imported the immutable Queue anyways):

```suggestion
      chargingPricesMemory = Queue.empty[Double]
```